test/unit/errors/error-collector.test.js

```diff
@@ -52,74 +47,79 @@ function createBackgroundTransaction(agent) {
   return createTransaction(agent, null, false)
 }
 
-describe('Errors', function () {
+tap.test('Errors', (t) => {
   let agent = null
 
-  beforeEach(function () {
+  t.beforeEach(() => {
+    if (agent) {
```

Comment:
you're doing this in both the before and after each. you probably only want to do this in after each

Reply:
ok I see why you're doing this. it's a little messy and prob best to follow the patterns where every child suite calls a before each function.  Check out all the unit tests in `tests/unit/shim`